src/rgw/rgw_sync_module.cc

```diff
@@ -63,4 +63,7 @@ void rgw_register_sync_modules(RGWSyncModulesManager *modules_manager)
 
   RGWSyncModuleRef es_module(std::make_shared<RGWElasticSyncModule>());
   modules_manager->register_module("elasticsearch", es_module);
+
+  RGWSyncModuleRef cloud_module(std::make_shared<RGWCloudSyncModule>());
+  modules_manager->register_module("cloud", cloud_module);
```

Comment:
There is one module in a rgw instance. If you want ufile & oss at the same time
,you must create two rgw instances